.cirrus.yml

```diff
@@ -91,7 +92,7 @@ osx_task:
         export PATH=$GOPATH/bin:$PATH
         brew update
         brew install gpgme go go-md2man
-        go install golang.org/x/lint/golint@latest
+        curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.52.2
```

Comment:
Consider setting/using an env-var for the version number.  You can do that up under the `Global variables used for all tasks` section.  That should be easier to maintain.

Alternatively, if parity with local development is important, consider doing the install and encoding the version in the `Makefile` instead.

Reply:
Thanks, not duplicating the version number is a great point.

I went with `make tools`, somewhat similar to what c/image does.